lib/bootstrap/environment.rb

```diff
@@ -64,11 +63,23 @@ def pattern_path(path)
 if $0 == __FILE__
   LogStash::Bundler.setup!({:without => [:build, :development]})
   require ARGV.shift
+
+  if settings = YAML.parse(IO.read("settings.yml"))
```

Comment:
Leaving a note here to make sure we read from a more robust path in a subsequent commit (maybe LS_HOME/settings.yml)